src/services/FileList.js

```diff
@@ -21,7 +21,7 @@
  */
 
 import client from './DavClient'
-import { genFileInfo } from '../utils/fileUtils'
+import { genFileInfo, encodeFilePath } from '../utils/fileUtils'
```

Comment:
you may also use https://nextcloud.github.io/nextcloud-paths/globals.html#encodepath :)

Reply:
:O